quirks/html-fills-viewport-quirk-ref.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<body style="width: 100vw; height: 100vw; margin: 0; padding: 10px; box-sizing: border-box">
```

Comment:
The height should be `100vh`

```suggestion
<body style="width: 100vw; height: 100vh; margin: 0; padding: 10px; box-sizing: border-box">
```